Library/Homebrew/cmd/outdated.rb

```diff
@@ -37,4 +33,30 @@ def outdated_brews(formulae)
       end
     end.compact
   end
+
+  def formulae_to_check
+    ARGV.resolved_formulae.any? ? ARGV.resolved_formulae : Formula.installed
+  end
+
+  def print_outdated
+    outdated_brews(formulae_to_check) do |f, versions|
+      if ($stdout.tty? || ARGV.verbose?) && !ARGV.flag?("--quiet")
+        puts "#{f.full_name} (#{versions*', '} < #{f.pkg_version})"
+      else
+        puts f.full_name
+      end
+    end
+  end
+
+  def print_outdated_json
+    json = []
+    outdated = outdated_brews(formulae_to_check) do |f, versions|
+      json << {name: f.name,
+               versions: versions.collect(&:to_s),
+               pkg_version: f.pkg_version.to_s}
```

Comment:
Alright, I'll handle the 1.8 hash syntax.

Should the `pkg_version` be `current_version` *everywhere* in this file? That seems like that's enough of a change to warrant a second commit.